src/providers/pronouns/UserPronouns.cpp

```diff
@@ -0,0 +1,24 @@
+#include "providers/pronouns/UserPronouns.hpp"
+
+#include <QString>
+
+#include <optional>
+
+namespace chatterino::pronouns {
+
+UserPronouns::UserPronouns(QString pronouns)
+    : representation{pronouns.length() != 0 ? std::move(pronouns) : QString()}
```

Comment:
```suggestion
    : representation{pronouns.isEmpty() ? std::move(pronouns) : QString()}
```

Although we don't use ternaries, I don't mind it here. If you want to get rid of this, make sure you use `isEmpty` in `UserPronouns::isUnspecified` and `UserPronouns::operator bool`. In Qt 6 `isEmpty` should be as fast as `isNull` (i.e. inlined). I don't really care about Qt 5.

Reply:
I applied the other changes, but this seems to break it for some reason. I'll investigate real quick